source/reference/minio-mc/mc-replicate-diff.rst

```diff
@@ -0,0 +1,116 @@
+.. _minio-mc-replicate-diff:
+
+=====================
+``mc replicate diff``
+=====================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc replicate diff
+
+Description
+-----------
+
+.. start-mc-replicate-diff-desc
+
+The :mc:`mc replicate diff` shows object versions that are not replicated, either generally for the bucket or to a specific remote bucket.
+
+.. end-mc-replicate-diff-desc
+
+Syntax
+------
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+      The following command shows unreplicated objects for the ``teamorange/projects`` prefix in the ``notes`` bucket for the ``myminio`` ALIAS:
```

Comment:
Just to confirm - should we just specify `--arn` here so users from the outset understand how to use this?

Chances are they do not intend to get the default status, given that they can get that with `mc replicate info` anyways.